src/components/SecondaryNav/SecondaryNav.story.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import SecondaryNav from './SecondaryNav';
+import SecondaryNavLink from './SecondaryNavLink';
+
+const stories = storiesOf('SecondaryNav', module);
+
+stories.addDecorator(withKnobs);
+
+stories.add('SecondaryNav', () => (
+  <SecondaryNav
+    buttons={[
```

Comment:
User knobs to allow users to add as many buttons as they want and to switch the active state of them